lib/improver/utilities/spatial.py

```diff
@@ -339,3 +340,79 @@ def process(self, cube):
         result_cube = check_cube_coordinates(cube, result_cube)
 
         return result_cube
+
+
+def lat_lon_determine(cube):
+    """
+    Test whether a diagnostic cube is on a latitude/longitude grid or uses an
+    alternative projection.
+
+    Args:
+        cube (iris.cube.Cube):
+            A diagnostic cube to examine for coordinate system.
+
+    Returns:
+        trg_crs (cartopy.crs/None):
+            Coordinate system of the diagnostic cube in a cartopy format unless
+            it is already a latitude/longitude grid, in which case None is
+            returned.
+
+    """
+    trg_crs = None
+    if (not cube.coord(axis='x').name() == 'longitude' or
+            not cube.coord(axis='y').name() == 'latitude'):
+        trg_crs = cube.coord_system().as_cartopy_crs()
+    return trg_crs
+
+
+def lat_lon_transform(trg_crs, latitude, longitude):
+    """
+    Transforms latitude/longitude coordinate pairs from a latitude/longitude
+    grid into an alternative projection defined by trg_crs.
+
+    Args:
+        trg_crs (cartopy.crs/None):
+            Target coordinate system in cartopy format or None.
+
+        latitude (float):
+            Latitude coordinate.
+
+        longitude (float):
+            Longitude coordinate.
+
+    Returns:
+        x, y (floats):
+            Longitude and latitude transformed into the target coordinate
+            system.
+
+    """
+    if trg_crs is None:
```

Comment:
I think it's slightly unusual that this function is expecting a target coordinate system of None to be passed into the function, as I would expect this more, if the the function had been defined as:
```python
def lat_lon_transform(latitude, longitude, trg_crs=None):
```
Having said that, this isn't a significant issue, so if we think a different behaviour is more desirable is the future, potentially when we're making use of this in other places, then we can modify the function then.

Reply:
I think that this is because lat_lon_determine returns None if the cube is in lat/lon or trg_crs if not so the two functions can be easily chained together to convert lat/lon to the target coordinate system of the cube catching the case where the cube is already in lat/lon.